types/elsys/elt-2/meta.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "ELT-2",
-  "version": "1.0.0",
+  "version": "2.3.5",
```

Comment:
@sevyakenza This is the version of our script, not the version of the sensor. Should be 1.0.0